src/test/scala/nl.knaw.dans.easy.deposit/SubmitterSpec.scala

```diff
@@ -57,7 +57,7 @@ class SubmitterSpec extends TestSupportFixture with MockFactory {
       .getOrRecover(e => fail(s"could not get stateManager of test deposit $e"))
     addDoiToDepositProperties(getBag(depositDir))
 
-    createSubmitter(unrelatedGroup).submit(depositDir, stateManager, "fullName", testDir / "staged") should matchPattern {
+    createSubmitter(unrelatedGroup).submit(depositDir, stateManager, UserInfo(userMap), testDir / "staged") should matchPattern {
```

Comment:
Not sure, but now you're implicitly testing `UserInfo.apply` here as well? Shouldn't there be a `testUser` object around here instead and have separate tests (if they're not there yet) for `UserInfo.apply`?

Same below and in other files...